auth/grants.go

```diff
@@ -93,6 +95,20 @@ func (c *ClaimGrants) Clone() *ClaimGrants {
 	return &clone
 }
 
+func (c *ClaimGrants) MarshalLogObject(e zapcore.ObjectEncoder) error {
+	if c == nil {
+		return nil
+	}
+
+	e.AddString("Identity", c.Identity)
+	e.AddString("Kind", c.Kind)
+	e.AddObject("Video", c.Video)
+	e.AddObject("SIP", c.SIP)
+	e.AddObject("RoomConfig", logger.Proto((*livekit.RoomConfiguration)(c.RoomConfig)))
+	e.AddString("RoomPreset", c.RoomPreset)
```

Comment:
Not logging `Sha256` as it is not useful for human consumption.
Not logging `Metadata` and `Attributes` due to PII.